package.json

```diff
@@ -14,6 +14,7 @@
     "babylon": "^6.17.4",
     "babylon-walk": "^1.0.2",
     "browser-resolve": "^1.11.2",
+    "bsb-js": "^1.0.1",
```

Comment:
Let's not take this on as a dependency - people can install it locally in their projects. can you move it to a dev dep for the tests and use `localRequire` below?

Reply:
bsb-js is a thin wrapper over finding and running a BuckleScript executable, kind of like the logic in the TS asset. I developed he package for use cases like this, adding Reason support easily to a new bundler. 

This doesn’t actually depend on BuckleScript, so users would still have to install that themselves. Requiring users to install it themselves wouldn’t make that much sense because it’s a package meant to be used by tooling. 

With this being the case, it is ok to add it as a dependency? If it helps at all, I’m the maintainer of that package.

Reply:
After #306 lands this won't be such a big deal since deps like this will be automatically installed by parcel as needed.

Reply:
Even so, `bsb-js` is still a pretty small dependency to add. It doesn't include the entire compiler toolchain and has no dependencies itself. The package should only add [~2.7k](https://bundlephobia.com/result?p=bsb-js@1.0.1)

Reply:
It's a slippery slope I think. We don't do it for other asset types, so I don't want to start now.